src/components/modals/EncryptPrivateKeyModal.vue

```diff
@@ -0,0 +1,129 @@
+<template>
+    <modal-no-teleport ref="baseModal" :action="action" @cancel="close(action.CANCEL)">
+        <template #header>
+            <p class="text-2xl text-gray-900">Choose Encryption Password</p>
+        </template>
+        <div class="flex flex-col">
+            <div class="my-2 w-full flex justify-center">
+                <div class="fa-stack fa-2x">
+                    <i class="fas fa-circle fa-stack-2x text-blue-200" />
+                    <i class="fas fa-exclamation fa-stack-1x text-blue-700" />
+                </div>
+            </div>
+            <p>
+                Please choose a password to encrypt your private key, so it can be securely stored on our servers. Bad things will happen if
+                you lose this password as it <span class="text-blue-800 font-semibold">cannot be restored</span>.
```

Comment:
```suggestion
                Please choose a password to encrypt your private key, so it can be securely stored on our servers. Ensure that you do not lose this password as it <span class="text-blue-800 font-semibold">cannot be restored</span>.
```

"Bad things will happen" sounds like an omen or a subliminary threat by the mafia :D